src/OSPSuite.UI/Services/CsvSeparatorSelector.cs

```diff
@@ -0,0 +1,27 @@
+using OSPSuite.Infrastructure.Import.Services;
+using OSPSuite.Presentation.Core;
+using OSPSuite.Presentation.Presenters.Importer;
+
+namespace OSPSuite.UI.Services
+{
+   public class CsvSeparatorSelector : ICsvSeparatorSelector
+   {
+      private readonly IApplicationController _applicationController;
+
+      public CsvSeparatorSelector(IApplicationController applicationController)
+      {
+         _applicationController = applicationController;
+      }
+
+      public char GetCsvSeparator(string fileName)
+      {
+         using (var csvSeparatorPresenter = _applicationController.Start<ICsvSeparatorSelectorPresenter>())
+         {
+            csvSeparatorPresenter.SetFileName(fileName);
+
+            return csvSeparatorPresenter.Canceled() ? '\0' : csvSeparatorPresenter.GetCsvSeparator();
```

Comment:
this char is weird, especially becuase it leaks outside of the implementation. REturn a char? and if it's null, it's not valid

Reply:
done